examples/with-wallet-stamper/src/components/user.tsx

```diff
@@ -15,14 +15,26 @@ import { Wallet } from "@/lib/types";
 import Wallets from "./wallets";
 import { Input } from "./ui/input";
 
+import { useTurnkey as useReactTurnkey } from "@turnkey/sdk-react";
+
 const User: React.FC = () => {
+  const { turnkey, authIframeClient } = useReactTurnkey();
   const { user, createWallet, getWallets } = useTurnkey();
   const [wallets, setWallets] = useState<Wallet[]>([]);
   const [walletName, setWalletName] = useState<string>("");
 
+  // Ideally use something like SWR to handle this properly.
+  // Load wallets on mount
   useEffect(() => {
     getWallets().then((wallets) => setWallets(wallets));
-  }, [getWallets]);
+  }, []);
+
+  const handleCreateWallet = (walletName: string) => {
+    createWallet(walletName, () => {
+      getWallets().then((wallets) => setWallets(wallets));
```

Comment:
Passing a callback here. A bit of an antipattern, but just a quick solution until proper resource fetching is in place